### PR TITLE
uploaded:Assignment

### PR DESCRIPTION
**Pull request by shivani-pawbake**

### CODE REVIEW

**Review by pritikekan — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Amit727619 — APPROVED** [review]

**Review by anuja1603 — APPROVED** [review]

**Review by pooja-nagare — APPROVED** [review]

File: Shivani/HTML/assign2/que.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<div><p style="font-size:13px;"><i>This article is about the company.</i></p>
+<h2>From the garage to the Googleplex</h2>
+<p>The Google story begins in 1995 at <strong><b>Stanford University.</b></strong><br>
+<strong><i>Larry Page</i></strong> was considering Stanford for grad school and <strong><i>Sergey Brin,</i></strong> a student there, was assigned to show him around.</p> 
+<p>By some accounts, thet disagreed about nearly everything during that first meeting. but by the following year they struck a <mark>partenrship.</mark><br>Working from their dorm rooms, they built a search engine that used links to determine the importance of individual page on the <span style="color:blue;">World Wide Web.</span></p>
+<p style="font-size:13px;"> They called this search engine <i><mark>Backrub.</mark></i></p>
+<hr>
+<h2>Initial public offering</h2>
+<p>Google's initial public offering (IPO) took place five years later,on 19<sup>th</sup> August 2004.<br> At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for<del>30</del> <ins>20</ins> years, until the 2024.</p>
+<h2 style="color:green;"><center>Thank You!!</center></h2></div>
```
Comment (shivnanda6272) [inline Shivani/HTML/assign2/que.html:17]: we don't use <center>as a tag, it is older version

Comment (shivnanda6272) [inline Shivani/HTML/assign2/que.html]: we don't use center as a tag.it is older version.

File: Shivani/HTML/assign3/2/Mean.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <div style="display: flex;">
+        <div><img src="download.jpeg" style="height:500px; width:350px; margin:20px; padding:10px;" alt="Nature1"></div>
+        <div><img src="download.jpeg" style="height:500px; width:350px; margin:20px; padding:10px;" alt="Nature2"></div>
+        <div><img src="download.jpeg" style="height:500px; width:350px; margin:20px; padding:10px;" alt="Nature3"></div>
+        <div><img src="download.jpeg" style="height:500px; width:350px; margin:20px; padding:10px;" alt="Nature4"></div>
+
+
+    </div>
+</body>
+</html>
```
Comment (shivnanda6272) [inline Shivani/HTML/assign3/2/Mean.html:43]: add new line

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]
changes request

**Review by shubhamdalal49 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Amit727619 — APPROVED** [review]

**Review by shivnanda6272 — APPROVED** [review]

**Review by shivnanda6272 — APPROVED** [review]

**Review by shivnanda6272 — APPROVED** [review]

**Review by shivnanda6272 — APPROVED** [review]

**Pull request merged**